<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SS_form_2</class>
 <widget class="QWidget" name="SS_form_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form2</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Text:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_inputText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>271</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_submitText">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
